tutorbackup/templates/backup/build/backup/backup_services.py

```diff
@@ -152,14 +151,13 @@ def archive(paths):
 
 
 def upload_to_s3():
-    from s3_client import S3_CLIENT, IntegrityError
+    from s3_client import S3_CLIENT, IntegrityError, calculate_checksum
 
     bucket = ENV['S3_BUCKET_NAME']
     file_name = TARFILE
 
     logger.info(f"Uploading {file_name} to S3 bucket {bucket}")
```

Comment:
Maybe you'd like to move this message down until after the checksum calculation, and add another log message here indicating that we're calculating the checksum.

Reply:
Yes, I'll do that.